### PR TITLE
Warrior lunge range qol

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Prevent from lunging if the target is too far

## Changelog
:cl:
qol: Prevent warrior from lunging if the target is too far
/:cl:

### CODE REVIEW

**Comment by Lefunny** [comment]
pov: instead of fixing how horrible warrior lunge is they nerf it even more instead

File: code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm [hunk 1]
```diff
@@ -73,6 +73,11 @@
 	if(!.)
 		return FALSE
 
+	if(get_dist_euclide_square(A, owner) > 36)
+		if(!silent)
+			to_chat(owner, span_xenonotice("You are too far!"))
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/warrior/abilities_warrior.dm:78]: shouldnt this 36 be a define

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Comment by Naaanii** [comment]
Will it also prevent the lunge into corners but missing the target?

**Comment by BraveMole** [comment]
> Will it also prevent the lunge into corners but missing the target?

no

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**